### PR TITLE
Fix spinner

### PR DESCRIPTION
**Pull request by NacymeG**

### CODE REVIEW

**Review by Caroline-CD — DISMISSED** [review]
Ok for modifications

**Review by Caroline-CD — DISMISSED** [review]
News modifications are ok

**Comment by LoicBrassart** [comment]
Please fix and reassign me

**Review by Caroline-CD — DISMISSED** [review]

File: src/Components/Footer/Footer.jsx [hunk 1]
```diff
@@ -3,7 +3,7 @@ import './Footer.css';
 import { Link } from 'react-router-dom';
 import { Popup } from 'semantic-ui-react';
 import FooterModal from './FooterModal';
-import { Link } from 'react-router-dom';
+
```
Comment (LoicBrassart) [inline src/Components/Footer/Footer.jsx]: Useless empty lines + I reiterate : what's happening with your Prettier ? Fix this, on P3 I'll refuse any unlintd PR without reading it

File: src/Containers/FilterOption.jsx [hunk 2]
```diff
@@ -56,76 +56,112 @@ const FilterOption = () => {
...
+          <p className="pTitle"> Type : </p>
+          <select
+            name="types"
+            className="typeSelect"
+            onChange={HandlerChangeTypeValue}
+          >
+            <option value="">Select a Type</option>
+            {listTypes.map((type) => {
+              return (
+                <>
```
Comment (LoicBrassart) [inline src/Containers/FilterOption.jsx]: Useless Fragment

File: src/Containers/FilterOption.jsx [hunk 3]
```diff
@@ -56,76 +56,112 @@ const FilterOption = () => {
...
+          <p className="pTitle">Sets : </p>
+          <select
+            name="sets"
+            className="setsSelect"
+            onChange={HandlerChangeSetValue}
+          >
+            <option value="">Select a Set</option>
+            {listSets.map((set) => {
+              return (
+                <>
```
Comment (LoicBrassart) [inline src/Containers/FilterOption.jsx]: Useless Fragment

**Review by Caroline-CD — APPROVED** [review]

**Review by LoicBrassart — APPROVED** [review]
Saw no real Evil, but plz fix Prettier asap

**Pull request merged**